build.gradle

```diff
@@ -58,6 +58,10 @@ subprojects {
     GUAVA: '28.1-jre',
     JSR305: '3.0.2', // transitively pulled in by GUAVA
 
+    // for Build Plan and Jib Plugins Extension API
+    BUILD_PLAN: '0.1.0',
```

Comment:
I wonder if we can use platform here in the future. Lemme take a look, this is fine for now though.